src/components/simulation/ExplanationSelectionObstacles.vue

```diff
@@ -2,10 +2,21 @@
 import step2 from '@/assets/illustrations/Steps.jpg'
```

Comment:
The component ExplanationSelectionObstacles.vue is only for this part:
![image](https://user-images.githubusercontent.com/121813008/225059002-6f78715e-c64d-44c2-bcdc-7026232f174a.png)
To manage the substep of step 2 can you create an another component please